### PR TITLE
Save only modifed form properties

### PR DESCRIPTION
**Pull request by uidoyen**

### CODE REVIEW

**Comment by indraraj** [comment]
Tested locally, seems to be working fine. Can you also please convert the properties `Key` (with `.` separated instead of `_`) before passing it to the host UI.

**Comment by uidoyen** [comment]
> Tested locally, seems to be working fine. Can you also please convert the properties `Key` (with `.` separated instead of `_`) before passing it to the host UI.

Modified accordingly!

**Review by indraraj — CHANGES_REQUESTED** [review]

File: ui/packages/ui/src/app/pages/createConnector/noValidation/debezium/Properties.tsx [hunk 1]
```diff
@@ -97,16 +99,22 @@ export const Properties: React.FC<IPropertiesProps> = (props) => {
...
+
     const updatedConfiguration = new Map([
       ...Array.from(configCopy.entries()),
       ...Array.from(formValues.entries()),
     ]);
-    props.onChange(updatedConfiguration, isFormValid(updatedConfiguration));
-
+    const finalConfiguration = new Map();
+    updatedConfiguration.forEach((value: any, key:any) => {
+      finalConfiguration.set(key.replace(/_/g, "."), value)
```
[the remark this answers is not in the document]
Comment (uidoyen) [reply ui/packages/ui/src/app/pages/createConnector/noValidation/debezium/Properties.tsx:115]: Updated accodingly. Thanks for the catch.

**Review by indraraj — APPROVED** [review]

**Review by mdrillin — APPROVED** [review]
lgtm

**Pull request merged**